projects/transfer-journey/src/lib/components/make-transfer-summary/make-transfer-summary.component.html

```diff
@@ -2,11 +2,11 @@
   <p i18n="summary heading label@@transfer.summary.heading.label">Are you sure to send out this transfer?</p>
   <strong i18n="summary account description label@@transfer.summary.account.description.label">it will be send to account:</strong>
   <br />
-  {{transfer?.toAccount}}
+  <span class="item-value">{{transfer?.toAccount}}</span>
```

Comment:
Let's use `data-roles` instead? and I would vote for naming them with descriptive names

Reply:
I am not a fan of data-attributes but I see that most widgets use that. So now I am using data-role